src/main/java/seedu/address/logic/commands/DeleteBookCommand.java

```diff
@@ -68,7 +68,7 @@ public boolean equals(Object other) {
         }
 
         // instanceof handles nulls
-        if (!(other instanceof DeleteCommand)) {
+        if (!(other instanceof DeleteBookCommand)) {
```

Comment:
Changed from DeleteCommand to DeleteBookCommand.

This allows DeleteBookCommandTest and DeleteBookCommandParserTest to pass.